src/extensions/default/PrefsCodeHints/main.js

```diff
@@ -38,11 +38,24 @@ define(function (require, exports, module) {
         JSONUtils           = brackets.getModule("language/JSONUtils"),
         Strings             = brackets.getModule("strings"),
         ThemeManager        = brackets.getModule("view/ThemeManager"),
+        CodeInspection      = brackets.getModule("language/CodeInspection"),
         _                   = brackets.getModule("thirdparty/lodash"),
-        data                = JSON.parse(require("text!data.json")),
+        languages           = LanguageManager.getLanguages(),
```

Comment:
@MarcelGerber I gave this a shot but since `statusChange` event is fired before the theme gets registered, `ThemeManager.getAllThemes()` still provides the old list of themes. But it works for getting code hints from newly installed extensions.